lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/impl/MultiObserverImpl.java

```diff
@@ -63,8 +62,7 @@ private void initialize() {
    * @param <TEvent> The type of the event being processed.
    */
   private <TEvent> void unimplemented(final long identifier, final TEvent event) {
-    LOG.log(Level.INFO, "Unimplemented event: [" + Long.toString(identifier) +"]"
-                        + event.getClass().getName());
+    LOG.log(Level.INFO, "Unimplemented event: [" + identifier +"]" + event.getClass().getName());
```

Comment:
`LOG.log(Level.WARN, "Unimplemented event: [{0}]: {1}", new String[] {String.valueOf(identifier), event.getClass().getName()});` 
String interpolation does not occur in this case and String.valueOf(identifier) is required.